core/domain/user_jobs_continuous.py

```diff
@@ -50,275 +47,6 @@ class RecentUpdatesRealtimeModel(
     pass
 
 
-class DashboardRecentUpdatesAggregator(jobs.BaseContinuousComputationManager):
-    """A continuous-computation job that computes a list of recent updates of
-    explorations and feedback threads to show on a user's dashboard.
-
-    This job does not have a working realtime component: the
-    RecentUpdatesRealtimeModel does nothing. There will be a delay in
-    propagating new updates to the dashboard; the length of the delay will be
-    approximately the time it takes a batch job to run.
-    """
-
-    @classmethod
-    def get_event_types_listened_to(cls):
-        """Returns a list of event types that this class subscribes to."""
-        return []
-
-    @classmethod
-    def _get_realtime_datastore_class(cls):
-        return RecentUpdatesRealtimeModel
```

Comment:
Can we remove the definition of `RecentUpdatesRealtimeModel`?

Reply:
I think its not necessary, not used anywhere. Will remove it

Reply:
done